### PR TITLE
admin settings for alermanager domain + link to alertmanager ui

### LINKED ISSUE #3280: Add link to Alertmanager Component
Add a link that leads to the external Alertmanager UI in the "Alertmanager Configuration" tab at the bottom of the cluster details view.
URL looks like this: `https://<common-domain>/<cluster-id>/` 
`common-domain` is specified in `mlaAlertmanagerDomain` in the admin settings

image

### PR DESCRIPTION
**Pull request by kgroschoff**
### What this PR does / why we need it
Added admin settings for alermanager domain & a link to alertmanager ui.

### Which issue(s) this PR fixes

Fixes #3280

### Release note

```release-note
Added admin settings for alermanager domain and the link to alertmanager ui.
```

### CODE REVIEW

**Comment by kgroschoff** [comment]
@maciaszczykm / @floreks PTAL

File: src/app/cluster/cluster-details/alertmanager-config/component.ts [hunk 1]
```diff
@@ -49,6 +59,18 @@ export class AlertmanagerConfigComponent implements OnDestroy {
     return _.isEmpty(this.alertmanagerConfig) && !this.isClusterRunning;
   }
 
+  displayAlertmanagerUILink(): boolean {
```
Comment (maciaszczykm) [inline src/app/cluster/cluster-details/alertmanager-config/component.ts:62]: Method name doesn't describe well what it actually does. Perhaps add 'should' at the beginning?

Comment (kgroschoff) [reply src/app/cluster/cluster-details/alertmanager-config/component.ts]: sure. Done.

File: src/app/cluster/cluster-details/alertmanager-config/component.ts [hunk 2]
```diff
@@ -9,37 +9,47 @@
...
     private readonly _matDialog: MatDialog,
     private readonly _mlaService: MLAService,
-    private readonly _notificationService: NotificationService
+    private readonly _notificationService: NotificationService,
+    private readonly _settingsService: SettingsService
   ) {}
 
+  ngOnInit(): void {
+    this._settingsService.adminSettings.pipe(takeUntil(this._unsubscribe)).subscribe(settings => {
+      this._settings = settings;
```
Comment (maciaszczykm) [inline src/app/cluster/cluster-details/alertmanager-config/component.ts:49]: nit: In case of the one liners you can avoid using `{}`.

Comment (kgroschoff) [reply src/app/cluster/cluster-details/alertmanager-config/component.ts]: done :)

**Pull request merged**